internal/static/i18n/zh.yaml

```diff
@@ -0,0 +1,900 @@
+  OIDCSettings:
+    NotFound: OIDC 配置未找到
+    AlreadyExists: OIDC 配置已存在
+  SecretGenerator:
+    AlreadyExists: 秘密生成器已经存在
+    TypeMissing: 缺少秘钥生成器类型
+    NotFound: 未找到秘钥生成器
+  SMSConfig:
+    NotFound: 未找到 SMS 配置
+    AlreadyActive: SMS 配置已启用
+    AlreadyDeactivated: SMS 配置已停用
+  SMTPConfig:
+    NotFound: 未找到 SMTP 配置
+    AlreadyExists: SMTP 配置已存在
+    SenderAdressNotCustomDomain: 发件人地址必须在在实例的域名设置中验证。
+  Notification:
+    NoDomain: 未找到对应的域名
+  User:
+    NotFound: 找不到用户
+    AlreadyExists: 用户已存在
+    NotFoundOnOrg: 在所选组织中找不到用户
+    NotAllowedOrg: 用户不是所需组织的成员
+    UserIDMissing: 缺少用户 ID
+    DomainPolicyNil: 组织策略为空
+    EmailAsUsernameNotAllowed: 电子邮件不允许作为用户名
+    Invalid: 用户数据无效
+    DomainNotAllowedAsUsername: 域已保存，但无法使用
+    AlreadyInactive: 用户已处于停用状态
+    NotInactive: 用户未处于停用状态
+    CantDeactivateInitial: 处于初始状态的用户只能删除不能停用
+    ShouldBeActiveOrInitial: 用户不是处于启用的的或初始化的
+    AlreadyInitialised: 用户已经初始化
+    NotInitialised: 用户尚未初始化
+    NotLocked: 用户未锁定
+    NoChanges: 未发现任何更改
+    InitCodeNotFound: 未找到初始化验证码
+    UsernameNotChanged: 用户名未更改
+    Profile:
+      NotFound: 未找到个人资料
+      NotChanged: 个人资料未更改
+      Invalid: 个人资料数据无效
+    Email:
+      NotFound: 电子邮件没有找到
+      Invalid: 电子邮件无效
+      AlreadyVerified: 电子邮件已经过验证
+      NotChanged: 电子邮件未更改
+    Phone:
+      NotFound: 手机号码未找到
+      Invalid: 手机号码无效
+      AlreadyVerified: 手机号码已经验证
+    Address:
+      NotFound: 找不到地址
+      NotChanged: 地址没有改变
+    Machine:
+      Key:
+        NotFound: 未找到机器密钥
+    PAT:
+      NotFound: 未找到个人访问令牌
+    NotHuman: 用户必须是个人
+    NotMachine: 用户必须是技术人员
+    WrongType: 此用户类型不允许
+    NotAllowedToLink: 不允许使用外部身份提供者登录并注册用户
+    Username:
+      AlreadyExists: 用户名已被使用
+      Reserved: 用户名已被使用
+    Code:
+      Empty: 验证码为空
+      NotFound: 验证码不存在
+      Expired: 验证码已过期
+      GeneratorAlgNotSupported: 不支持的生成器算法
+    Password:
+      NotFound: 未找到密码
+      Empty: 密码为空
+      Invalid: 密码无效
+      NotSet: 用户未设置密码
+    PasswordComplexityPolicy:
+      NotFound: 未找到密码策略
+      MinLength: 密码太短
+      MinLengthNotAllowed: 给定的最小长度不被允许
+      HasLower: 密码必须包含小写
+      HasUpper: 密码必须包含大写
+      HasNumber: 密码必须包含数字
+      HasSymbol: 密码必须包含符号
+    ExternalIDP:
+      Invalid: 外部 IDP 无效
+      IDPConfigNotExisting: IDP 提供者对此组织无效
+      NotAllowed: 此组织不允许外部 IDP
+      MinimumExternalIDPNeeded: 必须添加至少一个 IDP
+      AlreadyExists: 外部 IDP 已存在
+      NotFound: 未找到外部 IDP
+    MFA:
+      OTP:
+        AlreadyReady: OTP (一次性密码) 已经设置好了
+        NotExisting: OTP (一次性密码) 不存在
+        NotReady: OTP (一次性密码) 还没准备好
+        InvalidCode: 无效的验证码
+      U2F:
+        NotExisting: U2F 不存在
+      Passwordless:
+        NotExisting: 未设置无密码登录
+    WebAuthN:
+      NotFound: 找不到 WebAuthN 令牌
+      BeginRegisterFailed: WebAuthN 注册失败
+      MarshalError: 编组数据错误
+      ErrorOnParseCredential: 解析凭证数据时出错
+      CreateCredentialFailed: 创建凭据时出错
+      BeginLoginFailed: WebAuthN 登录失败
+      ValidateLoginFailed: 验证登录凭据时出错
+      CloneWarning: 凭证可能被克隆
+    RefreshToken:
+      Invalid: Refresh Token 无效
```

Comment:
In some local software, similar computer proper nouns are not translated, because users do not know what they are after translation, and it is difficult for maintenance personnel to accurately locate them.

Reply:
thanks for clarification :)